tests/utils/helpers.ts

```diff
@@ -0,0 +1,49 @@
+import { Build, BuildStatus, VisualApi } from '@saucelabs/visual';
+
+export const RE_VISUAL_BUILD_LINK =
+  /https:\/\/app\.(([a-z0-9-]+)\.)?saucelabs\.(com|net)\/visual\/builds\/([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})/g;
+export const RE_VISUAL_BUILD_ID =
+  /([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})/g;
+
+export const SAUCE_VISUAL_PROJECT = 'E2E test project';
+export const SAUCE_VISUAL_BRANCH = 'E2E test branch';
+export const SAUCE_VISUAL_DEFAULT_BRANCH = 'E2E test default branch';
+export const SAUCE_VISUAL_BUILD_NAME = 'E2E test build name';
```

Comment:
Should we add some randomness here ? Like jobId or pipeline Id from Github ?

Reply:
I've included a random ID generated by crypto module here. Fetching GitHub job ID is not so straightforward (not sure why)